src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -9,6 +9,18 @@ public class Hardware {
 
 	// climb [20 - 29]
 
+	// shooter [30 - 39]
+
```

Comment:
Delete, and move the below intake section after the launcher section below (to keep the numbers in order)